src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -794,6 +791,57 @@ public async Task GoAwayFrame_AllPendingStreamsValid_RequestsSucceedAndConnectio
             }
         }
 
+        [ConditionalFact(nameof(SupportsAlpn))]
+        public async Task GoAwayFrame_AbortAllPendingStreams_StreamFailWithExpectedException()
+        {
+            using (var server = Http2LoopbackServer.CreateServer())
+            using (var client = CreateHttpClient())
```

Comment:
nit: shouldn't these 'var' be explicit type based on our own rules for this?

Reply:
Yup, will fix.  I just copy and pasted this from the test above and didn't notice.